lib/event.rb

```diff
@@ -0,0 +1,27 @@
+require 'pry'
+require_relative './standard_deviation_module'
+
+class Event
+  include StandardDeviation
+
+  attr_reader :name,
+              :ages
+
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def max_age
+     ages.max
+  end
+
+  def min_age
+   ages.min
+  end
+
+  def average_age
+   ages.sum.to_f / ages.length
+  end
+
```

Comment:
I would change the name of the method in the module to `standard_deviation` since it would be nice if that module was general purpose, i.e. didn't just work for this class. You could then do something like this:

```ruby
def standard_deviation_age
  standard_deviation(@ages)
end
```

Reply:
Would I put that method in the event class or the module?